homework-submission/Week1/app.js

```diff
@@ -0,0 +1,89 @@
+// This component is to create todoList Table.
+class Table extends React.Component {
+  render() {
+    const description = this.props.description;
```

Comment:
good destructoring, but you can do it all at once with `const { description, foo, bar } = this.props;`